### ISSUE TITLE
Banner with ads in the latest versions of the client

### ISSUE THREAD

**Issue by amd64fox** [mrpond/BlockTheSpot#223]
I get such banners with ads in `1.1.67.586` - `1.1.68.632`

Basically, they pop up immediately after the client is launched.

<details>
<summary><small>Screenshot 1</small></summary><p>

![Снимок экрана 2021-09-22 044334](https://user-images.githubusercontent.com/62529699/134294273-14dcf9ea-1488-40f9-8a10-556568180a23.jpg)

</details>
<details>
<summary><small>Screenshot 2</small></summary><p>

![Снимок экрана 2021-09-22 042536](https://user-images.githubusercontent.com/62529699/134294518-8bfee899-1749-482d-9e86-0026778f40c4.jpg)

</details>

<details>
<summary><small>Screenshot 3</small></summary><p>

![Снимок экрана 2021-09-01 115125](https://user-images.githubusercontent.com/62529699/133682799-587d2366-56a2-424d-a1a0-dd5ca7db7323.jpg)

</details>

<details>
<summary><small>Screenshot 4</small></summary><p>

![Снимок экрана 2021-09-16 232431](https://user-images.githubusercontent.com/62529699/133682608-9033894a-cfdf-4cf8-bbaf-d116a6cce7f1.jpg)

</details>

<details>
<summary><small>blockthespot_log.txt</small></summary><p>

LOG | 16-Sep-2021 13:17:08 - GetModuleInformation OK!
LOG | 16-Sep-2021 13:17:08 - main process - patch failed!
LOG | 16-Sep-2021 13:17:08 - main process 1.1.66.578.gc54d0f69-a - patch success!

</details>

**Comment by Daksh777** [comment]
I do not get any pop-up banner ads on Spotify for `1.1.68.628.geb44bd66` here.

**Comment by chris-kwl** [comment]
Had the same ads the last few days with 1.1.67, updated to 1.1.68.628.geb44bd66 a few hours ago and so far they are gone.
Using on both versions Daksh777s SpotifyNoPremium Theme and BlockThespot

But I noticed some placements in playlists:
![grafik](https://user-images.githubusercontent.com/59339711/133888139-18069c24-c048-44f4-9a71-0eef9f4f40dd.png)

**Comment by NastyJack** [comment]
Yep, same here. Getting popup ads on opening Spotify and these banners.
![image](https://user-images.githubusercontent.com/44137857/133954257-ce1c71a1-d0ad-417c-991a-7d291178f29e.png)

**Comment by idgas01** [comment]
You guys shouldn't waste your time with any of the BlockTheSpot releases in the last year.  They all have banner ads or large div containers that used to have the ads.  And there are a dozen or so complicated workarounds that take too much time and effort and they are only temporary before they break again.  The last time that BlockTheSpot genuinely worked was around November 18, 2020.  Yeah, everyone wants the latest version of the program because of the newest features and security fixes, but I spent the last year using an old version that performs perfectly and I never have to bother with updates, banners, and divs left behind by banners.  Just google "Spotify older versions" and install the 1.1.51 version.  Then go to the BlockTheSpot releases and go back a year and look for the 26FIX on November 18, 2020.  And you only need the chrome_elf, not the BlockTheSpot crap.  Don't bother posting here on their GitHub because none of these mods are going to help you.  They are just self important mannequins (fake people) who post a lot and talk out their behinds but they can't help you because they aren't the ones who develop the program.  And really the only people actually helping are the ones who come up with the workarounds, not the mods who just obstruct, give you useless information, have attitudes, close threads, etc.  Farts have more use than these mod clowns.

**Comment by chris-kwl** [comment]
For the pop-up / overlay `.GenericModal__overlay--afterOpen` is responsible with modify `opacity` from 1 to 0 it can be hidden, but the UI is not clickable. Perhaps `display: none` is working, but I could not test it so far since I get the overlay currently only once a day.

For the placement its `.sponsor-container` and it can be removed/hidden with `display: none`

@Daksh777 hope you have a look and add it to your theme :)

**Comment by Nuzair46** [comment]
@chris-kwl Sponsor banner is found only on some region and unrelated to this issue. Please open a new issue for this, I will add it for enhancement. Please copy the content in this https://github.com/mrpond/BlockTheSpot/issues/223#issuecomment-922267087 and open new issue for it so it can be managed separately.

@chris-kwl fixes to be used on SpotifyNoPremium by Daksh777 should be reported on the respective repo. Please don't use this repo to request features on somewhere else.

**Comment by Daksh777** [comment]
Hey @chris-kwlm, I'll add the sponsor container part to my repo but I'm not sure if the pop-up/overlay part is correct. Can you head over to my repo and create an issue please https://github.com/daksh777/spotifynopremium/issues

**Comment by SmartManoj** [comment]
Got ads
Spotify for Windows
1.1.68.632.g2b11de83

**Comment by chris-kwl** [comment]
Got the overlay displayed again today and tried it with `display: none` for `.GenericModal__overlay--afterOpen` and it worked for me. So adding this in `xpui.css` or remove the class should work.

**Comment by SmartManoj** [comment]
> Got ads
> Spotify for Windows
> 1.1.68.632.g2b11de83

Audio ads

**Comment by xerta555** [comment]
> 
> 
> Got ads
> Spotify for Windows
> 1.1.68.632.g2b11de83

> 
> 
> > Got ads
> > Spotify for Windows
> > 1.1.68.632.g2b11de83
> 
> Audio ads

I confirm, i get audio ads on this new version.
I get the same version from Spotify official website.
Most of the time for me, a simple reboot of Spotify client is enough.

If there is/are way(s) to get full logs to help debugging...

**Comment by amd64fox** [comment]
Hi, I made small changes in`xpui.js`, I have been testing for a day.

I still can't get the pop-up ad offer like before.

If there are those who wish, then join the test


- First, make a backup of your `xpui.spa` (AppData\Roaming\Spotify\Apps)
- Download modified [xpui.zip](https://github.com/mrpond/BlockTheSpot/files/7246755/xpui.zip) and rename it to `xpui.spa`
- Replace the original `xpiu.spa`

Share in the comments about the result, whether it is positive or negative.

**Comment by Nuzair46** [comment]
will be tracked on #227

**Comment by xerta555** [comment]
> 
> 
> Hi, I made small changes in`xpui.js`, I have been testing for a day.
> 
> I still can't get the pop-up ad offer like before.
> 
> If there are those who wish, then join the test
> 
> You need to check on the latest version of the client `1.1.68.632`
> 
>     * First, make a backup of your `xpui.spa` (AppData\Roaming\Spotify\Apps)
> 
>     * Download modified [xpui.zip](https://github.com/mrpond/BlockTheSpot/files/7246755/xpui.zip) and rename it to `xpui.spa`
> 
>     * Replace the original `xpiu.spa`
> 
> 
> Share in the comments about the result, whether it is positive or negative.

Hi

I would like to join the test, i use Spotify every day.

**Comment by amd64fox** [comment]
@xerta555 
Thanks, the BTS patch has already been fixed.
